ci/cloudbuild/builds/lib/features.sh

```diff
@@ -40,6 +40,8 @@ function features::always_build() {
     # Build our code with OpenTelemetry. This feature includes both the client
     # library instrumentation, and the GCP exporters.
     experimental-opentelemetry
+    # Enable storage-grpc in most builds.
```

Comment:
Is this something you would have changed to "storage_grpc" in #12686?

Reply:
Right you are. Fixed.